src/misc/dbg_helpers.cpp

```diff
@@ -62,10 +59,7 @@ std::string ValueStr(SignalType t)
 /** Translate TileIndex into string. */
 std::string TileStr(TileIndex tile)
 {
-	std::stringstream ss;
-	ss << "0x" << std::setfill('0') << std::setw(4) << std::hex << tile.base(); // 0x%04X
-	ss << " (" << TileX(tile) << ", " << TileY(tile) << ")";
-	return ss.str();
+	return fmt::format("{:#04X} ({}, {})", tile.base(), TileX(tile), TileY(tile));
```

Comment:
Having a lowercase `x` and upper case `A-F` makes it visually easier to read in my opinion.
With `"{:#04X}"` you'll get `0X005A`, with `"{:#04x}"` you'll get `0x005a`.

Currently it's common in the codebase to do (a variation of) `"0x{:X}"` when using `0x`.

Reply:
I also prefer 0xAB, fixed.